src/globalvars.h

```diff
@@ -9,15 +9,56 @@
 #include <unordered_set>
 
 #include "types.h"
+#include "util/result.h"
 
 namespace bpftrace {
 
+class NamedParamError : public ErrorInfo<NamedParamError> {
```

Comment:
Why not put this in the globalvars namespace? I'm just not clear if there's a specific reason for isolating this within the context of this header.